web/src/components/vis/CorrelationTile.vue

```diff
@@ -35,6 +35,10 @@ export default {
       correlation_methods,
       metaboliteFilter: null,
       metaboliteColor: null,
+      search: null, // nodes being searched for
+      currentUserInput: null, // the text currently typed in search box
+      searchBarResults: [], // current results from autocomplete search box
+      nodeCount: 0, // total number of nodes in graph
```

Comment:
seems like this would be good to refactor as a computed property.

It's perfectly valid to have computed properties that are derived from other computed properties.

Reply:
Also, these are helpful comments.  Documenting these really helps me confirm that the logic matches the intent.